database/seeds/OrganizationRoleTypeSeeder.php

```diff
@@ -12,33 +12,40 @@ class OrganizationRoleTypeSeeder extends Seeder
     public function run()
     {
         DB::table('organization_role_types')->insertOrIgnore([
+            'id' => 1,
             'name' => 'admin',
             'display_name' => 'Administrator'
         ]);
 
-        $courseCreatorId = DB::table('organization_role_types')->insertGetId([
+        DB::table('organization_role_types')->insertOrIgnore([
+            'id' => 2,
             'name' => 'course_creator',
             'display_name' => 'Course Creator'
         ]);
-
+        // This maybe a redundant query, but is okay
+        $courseCreator = DB::table('organization_role_types')->select('id')->where('name', 'course_creator')->first();
+        
         DB::table('organization_role_types')->insertOrIgnore([
+            'id' => 3,
             'name' => 'member',
             'display_name' => 'Member'
         ]);
 
         DB::table('organization_role_types')->insertOrIgnore([
+            'id' => 4,
             'name' => 'self_registered',
             'display_name' => 'Self Registered'
         ]);
 
-        $users = DB::table('users')->select('id')->get();
-
-        foreach ($users as $user) {
-            DB::table('organization_user_roles')->insertOrIgnore([
-                'organization_id' => 1,
-                'user_id' => $user->id,
-                'organization_role_type_id' => $courseCreatorId
-            ]);
+        if ($courseCreator) {
```

Comment:
This seeder was meant run one time only
with the introduction of multi tenancy using organizations,
it associate all the existing users with the default/first organization.

If this seeder will run now
it will wrongly associate all the users in the system belonging to different organizations
with the default/first organization.

Reply:
I have updated it so that the users are only patched if there is we have no records.